core/src/stack/inputs.rs

```diff
@@ -16,17 +16,33 @@ pub struct StackInputs {
 }
 
 impl StackInputs {
+    // CONSTANTS
+    // --------------------------------------------------------------------------------------------
+
+    pub const MAX_STACK_INPUTS_SIZE: usize = u16::MAX as usize;
```

Comment:
The naming is a bit redundant here - e.g., `StackInputs::MAX_STACK_INPUTS_SIZE`. I would probably change this to something like `MAX_LEN` and then it would be `StackInputs::MAX_LEN`.